src/main/java/seedu/clinic/logic/commands/HelpCommand.java

```diff
@@ -100,7 +99,7 @@ private String generateHelpGenericMessage() {
         String aboutListCommand = "list\nYou can list all suppliers and warehouses by using list command.";
         String aboutViewCommand = "view\nYou can view a specific warehouse or supplier using"
                 + " the view command.";
-        String aboutUpdateCommand = "update\nYou can update stock of a product in the warehouse by"
+        String aboutUpdateCommand = "update\nYou can update a product in a supplier or warehouse by"
```

Comment:
"product in a supplier" sounds a bit weird, maybe "a product managed by a supplier or warehouse" is better